components/brave_wallet/browser/brave_wallet_service.h

```diff
@@ -47,6 +47,8 @@ class BraveWalletService : public KeyedService,
       base::OnceCallback<void(bool, const std::string&, const std::string&)>;
   using AddSuggestTokenCallback =
       base::OnceCallback<void(bool, mojom::ProviderError, const std::string&)>;
+  using AddGetEncryptionPublicKeyCallback = base::OnceCallback<
```

Comment:
This should be `GetEncryptionPublicKeyCallback` and it should be available in generated `brave_wallet.mojom.h` so we don't need to redefine it so just use `mojom::BraveWalletProvider::GetEncryptionPublicKeyCallback`

Reply:
This was not needed anymore after the rebase, so it was removed.

Reply:
Fixed in 96ef1be1a6b706d017c8e86f8c4e0175f66d06b7